h/static/scripts/bridge.coffee

```diff
@@ -118,7 +118,22 @@ class Bridge
       options = $.extend {}, options, {origin: '*'}
     channel = Channel.build(options)
 
+class BridgeFactory
+
+  constructor: ->
+
+  _createBridge: ->
+    # Set up the bridge plugin, which bridges the main annotation methods
+    # between the host page and the panel widget.
+    options =
+      scope: 'annotator:bridge'
+    @_bridge = new Bridge(options)
+
+   $get: ->
+     @_createBridge() unless @_bridge?
+     @_bridge
+
 if angular?
-  angular.module('h').value('Bridge', Bridge)
+  angular.module('h').provider('bridge', BridgeFactory)
```

Comment:
There's no reason to use a provider with such a simple thing. Just have a function `createBridge` that takes no injectables and use `.factory('bridge', createBridge)`.